dotCMS/src/main/java/com/dotmarketing/business/VersionableAPI.java

```diff
@@ -310,7 +307,7 @@
 	 * @throws DotStateException
 	 * @throws DotSecurityException
 	 */
-	public Date getLockedOn(Versionable ver) throws DotDataException, DotStateException,DotSecurityException;
+	public Optional<Date> getLockedOn(Versionable ver) throws DotDataException, DotStateException,DotSecurityException;
```

Comment:
Codacy found an issue: [Unnecessary modifier 'public' on method 'getLockedOn': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6214071)